### ISSUE TITLE
How large is the dataset?

### ISSUE THREAD

**Issue by greeneggsandyaml** [hirokatsukataoka16/FractalDB#1]
Hello, 

How large is the dataset (in GB), and how long does it take for you to generate it? I'm trying to reproduce your results, and generation is taking a super long time, even on a machine with 64 CPUs. 

Also, once you generated the dataset, how did you train your classification model? 

Best,

**Comment by hirokatsukataoka16** [comment]
Hi @greeneggsandyaml ,

Thanks for your comments.

[How large is the dataset?]
In our observation, FractalDB-1k (1k categories / 1k instances) is 13GB. You can download the dataset the linked page. 
https://hirokatsukataoka16.github.io/Pretraining-without-Natural-Images/#dataset

[how long does it take?]
In terms of the rendering time, we can render 1 or 2 days without fractal category search. Did you try to render the fractal images with multi-thread execution? You can use the command in the execution file ( https://github.com/hirokatsukataoka16/FractalDB/blob/main/exe.sh ) instead of single-thread programming as follows. 

```
# Multi-thread processing
for ((i=0 ; i<40 ; i++))
do
    python fractal_renderer/make_fractaldb.py \
        --load_root='./data/csv_rate'${fillrate}'_category'${numof_category}'_parallel/csv'${i} \
        --save_root='./data/FractalDB-'${numof_category} --image_size_x=${imagesize} --image_size_y=${imagesize} \
        --iteration=${numof_ite} --draw_type=${howto_draw} --weight_csv='./fractal_renderer/weights/weights_'${weight}'.csv' &
done
wait
```

[how did you train your classification model?]
We have assigned a simple CNN training which is similar to ImageNet training. However, the normalization is different from the standard mean pixel value. Especially in the ImageNet mean pixel value, we replaced from (123,117,104) to (127, 127, 127). At this moment, we are preparing to share our codes in pre-training/fine-tuning and their weights.

**Comment by greeneggsandyaml** [comment]
Hi @hirokatsukataoka16 ,

Thank you so much for your quick and comprehensive response! I really appreciate you answering my questions in detail. 

I was able to download the dataset from the link on your project page, and I'm currently exploring the dataset. I look forward to seeing your training and fine-tuning code when it is released. 

I'm going to close this issue for now, and I will reply if I have additional questions. 

Best,

**Comment by hirokatsukataoka16** [comment]
Thanks, @greeneggsandyaml ,
Feel free to ask me again!

**Comment by hirokatsukataoka16** [comment]
Hi @greeneggsandyaml ,

I updated the repository including codes for pre-training and fine-tuning.

**Comment by greeneggsandyaml** [comment]
Brilliant, thank you!

**Comment by greeneggsandyaml** [comment]
Hello @hirokatsukataoka16,

I am had another quick question about finetuning -- how did you prepare the PascalVOC and Omniglot datasets? My understanding is that PascalVOC is usually in a multi-class classification setup and Omniglot is usually a few-shot learning setup. Did you make them into single-class classification tasks, and if so how exactly did you go about it?

Thank you so much again for the paper and repo! It's a really nice idea.
